### PR TITLE
Fix links

### PR DESCRIPTION
**Pull request by hargup**

### CODE REVIEW

**Comment by hargup** [comment]
Ping @athityakumar @DefCon-007 

This PR also fixes PEP8 errors, PEP8 is the standard python style guide https://www.python.org/dev/peps/pep-0008/. I also noticed that there was inconsistent use of `tabs` and `spaces` by convention, this doesn't mean you have to press the space key four times, you can configure your text editor to convert a tab key to four spaces.

**Comment by hargup** [comment]
Another thing is I've moved the code to enable links to `frontend.py`. `fbscrapper.py` should deal with purely downloading of raw data into jsons. Everything else goes to `frontend.py`. This is being done for two reasons:
1. In future we can have multiple frontends, calendar is also a kind of frontend, or we can have a terminal viewer. Separate frontend and scrapping code will make these changes easier.
2. Even if we never have another frontend, keeping these logic separate helps in debugging and maintenance of code. As problems of one "subsystem" don't leak into another "subsystem". I can read the json files to figure out where the error is, or I can use replace the existing jsons with some known correct jsons to see if rendering is correct or not.

This is following the concept of loose coupling which says that having a large system with many points of interactions, have many small systems interacting at small number of well defined points.

**Comment by hargup** [comment]
Also please remove the json files before testing this.

**Comment by DefCon-007** [comment]
@hargup  This particular post is taking me to the wrong url : 
screen shot 2016-08-24 at 2 38 51 am
URL : file:///Users/ayush/Dropbox/CODE/naarad/docs/www.venturesity.com/ibm

File: fbscraper.py [hunk 1]
```diff
@@ -62,44 +62,29 @@ def get_link(post_id):
...
-        if event['id'] in post_id :                
+    for event in all_events['data']:
+        if event['id'] in post_id:
             DateTime = prettify_date([{'created_time': event['start_time']}])
-            if 'description' in event.keys() :  #checking if the event have description 
-            	message = "%s \nDate : %s\nTime : %s\nVenue : %s "  %  (event['description'] ,DateTime[0]['real_time']  , DateTime[0]['real_date']  , event['place']['name'])
-            else :
-            	message = "%s \nDate : %s\nTime : %s\nVenue : %s "  %  (event['name'] ,DateTime[0]['real_date']  , DateTime[0]['real_time']  , event['place']['name']) 
+            if 'description' in event.keys():  # checking if the event have description
+                message = "%s \nDate : %s\nTime : %s\nVenue : %s " % (event['description'], DateTime[
```
Comment (raivivek) [inline fbscraper.py]: Might as well use Python 3 style formatting? @hargup

File: fbscraper.py [hunk 2]
```diff
@@ -148,26 +134,15 @@ def get_feed(page_id, pages=10):
         # determine the next page, until there isn't one
         try:
             next_page = feed['paging']['next']
-            next_search = re.search('.*(\&until=[0-9]+)', next_page, re.IGNORECASE)
+            next_search = re.search(
+                '.*(\&until=[0-9]+)', next_page, re.IGNORECASE)
```
Comment (raivivek) [inline fbscraper.py:147]: Seems like an unnecessary line break.

**Comment by athityakumar** [comment]
@DefCon-007 : Delete the existing json files and run the deploy script.

@hargup  : Thanks for the quick response and the PR! :smile:

**Comment by DefCon-007** [comment]
@athityakumar @hargup  : Ran the deploy script. Some new issues now. 
1. Some links are still not clickable. Post url : `https://www.facebook.com/149405445074499_1383394831675548`

screen shot 2016-08-24 at 12 59 50 pm
1. Images are not loading for any post. 
2. Some posts are completely blank. Post url : `https://www.facebook.com/741132932600291_1051679094879005`

screen shot 2016-08-24 at 1 02 59 pm

Will try to debug these.

**Comment by athityakumar** [comment]
Also, long links are yet to be shortened...

**Comment by athityakumar** [comment]
@hargup : I think these changes should be made in `fbscraper.py` and be stored into `feed.json` - because these changes are forgotten by naarad while fetching and showing search results, on searching for posts...

cc @vivekiitkgp @DefCon-007

**Comment by DefCon-007** [comment]
@athityakumar : We can keep the changes in the `frontend.py` and update the `feed.json` from the `frontend.py`. The benefit of this is we don't have to use `fbscrapper.py` again after making some changes. Scraping data takes quite time.

**Pull request merged**